Bottom: 412cc612c3773faafe6782b2f3ba29b69c23967c
Top:    ae1a449178a85ae5a43851edf52dc8759e6cedf0
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-04-01 10:25:28 -0400

Refresh of device-trees

---

diff --git a/sound/soc/codecs/uda1380.c b/sound/soc/codecs/uda1380.c
index 197463a..bddaf4d 100644
--- a/sound/soc/codecs/uda1380.c
+++ b/sound/soc/codecs/uda1380.c
@@ -161,13 +161,10 @@ static int uda1380_reset(struct snd_soc_codec *codec)
 		data[1] = 0;
 		data[2] = 0;
 
-		printk("JDS reset 1\n");
 		if ((temp = codec->hw_write(codec->control_data, data, 3)) != 3) {
-			printk("JDS reset 2 %d\n", temp);
 			dev_err(codec->dev, "%s: failed\n", __func__);
 			return -EIO;
 		}
-		printk("JDS reset 3\n");
 	}
 
 	return 0;
diff --git a/sound/soc/soc-core.c b/sound/soc/soc-core.c
index 306a577..92cee24 100644
--- a/sound/soc/soc-core.c
+++ b/sound/soc/soc-core.c
@@ -22,8 +22,6 @@
  *   o Support TDM on PCM and I2S
  */
 
-#define DEBUG
-
 #include <linux/module.h>
 #include <linux/moduleparam.h>
 #include <linux/init.h>
